src/app/containers/Text/index.test.jsx

```diff
@@ -26,7 +26,7 @@ describe('TextContainer', () => {
           '02',
           'This is another paragraph with some **bold** text.',
         ),
-        paragraphBlock('03', 'This is a paragraph with *italic* text.'),
+        paragraphBlock('03', 'This is a paragraph with __italic__ text.'),
         paragraphBlock(
```

Comment:
I think it would be useful to update tests to capture all four cases `*italic text*`, `_italic text_` and `__bold text__` `**bold text**`, since these are also valid markdown.

Reply:
To extend this thought, unless we filter the md we support shouldn't we test all possible md outputs? e.g. bullet points